diagrams-and-designs/Scenarios_For_SAR.md

```diff
@@ -0,0 +1,19 @@
+<strong><ins>Alma Baker, 34</ins></strong>
+
+<ins>Scenario: Making a Presentation on Scottish Law</ins>
+
+Alma is preparing a presentation for her upcoming seminar on Scottish law in the late Middle Ages. She logs into SAR to find records of inheritance disputes. She quickly navigates the intuitive interface and uses keywords related to her research topic. The advanced filtering options allow her to refine her search by date and type of record, which she finds helpful. After finding the relevant entries she wants to include in her presentation, she clicks the self-explanatory 'Add to selected' button. Once Alma has found some relevant entries, she goes to the 'View selected' page, which she easily found due to it being signposted as she started selecting entries. Then downloads them as a pdf and uses them in her presentation and closes down the application, happy to have found what she was looking for.
+
+
+<strong><ins>Calvin Woods, 38</ins></strong>
+
+<ins>Scenario: Discovering Family History</ins>
+
+Calvin's grandmother recently passed away, leaving him with a desire to explore his family's roots. He decides to use SAR to learn more about his heritage. Given his limited tech skills, Calvin appreciates the straightforward design of the application. He inputs his grandmother’s last name into the search bar and clicks enter. As he looks through the resutls he doesn't find what he's looking for so he begins an advanced search. He soon realizes that the application’s advanced search feature is overwhelming for him. He sticks to basic searches, this time inputting his grandmother's maiden name, and finds several results with his grandmother's maiden name. Though he can't fully grasp the context, he feels a connection to the past and prints the pages to discuss with his family.
```

Comment:
they could be worded as strengths of our desired system, rather than faults of the current one, cause that fits better with the current way things are worded

Reply:
Fixed spelling error, added dark mode into scenario.